worker/backend/spec/spec.go

```diff
@@ -109,19 +116,41 @@ func OciSpecBindMounts(bindMounts []garden.BindMount) (mounts []specs.Mount, err
 	return
 }
 
+// OciIDMappings provides the uid/gid mappings for user namespaces (if
+// necessary, based on `privileged`).
+//
+func OciIDMappings(privileged bool, max uint32) []specs.LinuxIDMapping {
+	if privileged {
```

Comment:
I guess we can't make this unconditional until we make `containerd` the sole driver for Linux, because right now it's the ATC and BaggageClaim that does the permission remapping based on whether the container is privileged. If we made this unconditional we would have to change all that.